internal/provider/generate.go

```diff
@@ -816,6 +883,29 @@ func (g *generator) renderStaticWebsite(providerSchema *tfjson.ProviderSchema) e
 				return nil
 			}
 			g.warnf("action entitled %q, or %q does not exist", shortName, resName)
+		case "list-resources/":
+			resSchema, resName := resourceSchema(providerSchema.ListResourceSchemas, shortName, relFile)
+			exampleFilePath := filepath.Join(g.ProviderExamplesDir(), "list-resources", resName, "list.tfquery.hcl")
```

Comment:
I'm not sure if this will cause problems (my initial thought is probably not? or at least the problems would be minor), but it's worth noting that without any other changes, these `tfquery.hcl` files will be put into `terraform` markdown code blocks and not `hcl` code blocks.

https://github.com/hashicorp/terraform-plugin-docs/blob/697541615c701bb6bef75ee214ed9d79664af126/internal/provider/template.go#L340
https://github.com/hashicorp/terraform-plugin-docs/blob/697541615c701bb6bef75ee214ed9d79664af126/internal/provider/template.go#L144-L153

Reply:
Oh okay cool, didn't know they will eventually be available in `.tf` files anyways. This seems like a non-issue then 👍🏻